runtime/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
log4j2 has many performance improvements compared to it's previous version.
http://www.infoq.com/news/2014/07/apache-log4j2

.properties file support has been removed in the new version. It supports only xml, json and yaml files.